client/src/containers/PlayersList/PlayersList.scss

```diff
@@ -0,0 +1,20 @@
+@import 'styles/variables';
+
+.PlayersList {
+  h2 {
+    margin-bottom: $spacing-small;
+  }
+
+  &__Grid {
+    display: flex;
+    flex-wrap: wrap;
+  }
+
+  &__GridItem {
+    width: 45%;
```

Comment:
no point using `flex-wrap` if the child element has relative width as the children will never wrap

Reply:
An alternative could be to use the `flex` property instead in combination with a reasonably set `min-width`.

```scss
&__GridItem {
  flex: 1 0 50%;
  min-width: 15rem;
}
```

Reply:
Okay I found a proper way to make it flexible so we don't need media queries. But we do need the `flex-wrap` and a `min-width`. We have 3 solutions to set the width:
- Create a new `$spacing-x` variable.
- Multiple the `$spacing-regular` variable.
- Don't use a variable at all: `15rem`.

I prefer the 2nd one except if we know we will use the new variable elsewhere.